content/articles/building-scalable-projects-using-the-multi-module-architecture-with-spring-boot-and-maven/index.md

```diff
@@ -0,0 +1,115 @@
+# Multi-Module Project Setup in Spring Boot
+
+A three-man organisation in 2019 grew to become a twelve-man team fourteen months later. The growth in team members was necessitated by the growth in the project. However, for them also came an increase in the cost of building, maintaining and scaling the project; they were running the microservices architecture.
+
+In this tutorial, we will be going through the multi-module architecture - which is an drastic improvement on the monolith approach, yet offering as much possibilities of the microservices architecture, while evading the huge demands of microservices. We will be building a model of a hospital management system, taking into account some of the departments that make for a successful medical application.
+
+## Table of contents
```

Comment:
###

Reply:
What would you have me do here? You didn't leave a comment on what to do

Reply:
I'm sorry. Must have been some kind of bug.

I want you to change the markdown header to '###'